### DISCUSSION TITLE
ASE Dependency

### DISCUSSION THREAD

**Discussion by 97gamjak** [MolarVerse/PQAnalysis#67] [Ideas]
After some thoughts i would really suggest that we try to introduce ASE dependencies only in the internal parts of the code. Meaning that all API functions are handled by the PQAnalysis to keep it as unified as possible. 
For example all reading and wroting stuff... and then internally if we need some functionality by ASE we build our *private* converter.

@galjos @stkroe

**Comment by galjos** [comment]
We could also add .exyz to @MolarVerse/pq. I think this would increase compatibility.

**Reply by galjos** [reply]
exyz with, maybe, ASE specific-comment section

**Reply by galjos** [reply]
Also, all old analysis tools will be rewritten and built into this project. There's no need to keep the compatibility with _thh_tools_ alive.

**Reply by galjos** [reply]
#stickToStandards
